### ISSUE TITLE
Apply to become a DPG

### ISSUE THREAD

**Issue by mgifford** [localgovdrupal/localgov#801]
I'd love for LocalGov Drupal to become a Digital Public Good

You appeared in this list here:
https://beeckcenter.georgetown.edu/report/landscape-scan-of-digital-public-goods-use-in-government/

Drupal already is one:
https://www.digitalpublicgoods.net/r/drupal

We applied to make DKAN one (and it is based on Drupal)
https://www.digitalpublicgoods.net/r/dkan

Take a look at:
https://www.digitalpublicgoods.net/digital-public-goods/submission-guide

**Comment by finnlewis** [comment]
Good thinking! @aaronhirtenstein mentioned that Rachel Norfolk also talked about this at some point. Let's do it!

**Comment by aaronhirtenstein** [comment]
Yes absolutely, thanks for the nudge @mgifford ! I've added this to agenda of next core team meeting 😄

**Comment by finnlewis** [comment]
@mgifford coming to this tomorrow? https://lu.ma/rkd8yhih

**Comment by mgifford** [comment]
I just registered and will make it if I can.

**Comment by markconroy** [comment]
I wrote a blog post about this about a month ago.

https://mark.ie/blog/localgov-drupal-lgd-a-digital-public-good-transforming-government-services/

Looks like lots of us are having similar thoughts.

**Comment by finnlewis** [comment]
@willguv let's prioritise this in the new year!

**Comment by willguv** [comment]
There needs to be push for governments to use DPGs - in the UK there would need an update to https://www.gov.uk/guidance/the-technology-code-of-practice 

I’ll find out who’s working on this

**Comment by aaronhirtenstein** [comment]
@willguv I think you were looking at this after the Dec meetup, did you get anywhere? Came up again at DrupalCamp England..

**Comment by willguv** [comment]
Hi Aaron, Finn and I filed it. Haven't heard anything back

**Comment by willguv** [comment]
Can you move to done in the 2025 priorities project and maybe close it please? Don't seem to be able to do this on my phone

**Comment by aaronhirtenstein** [comment]
it wasn't in there but I added it and then moved to done which has closed the issue.
